frontend/src/common/MarkdownDisplay.tsx

```diff
@@ -99,6 +100,24 @@ export default function MarkdownDisplay({ children }: MarkdownDisplayProps) {
             />
           ),
         },
+        code: {
+          component: ({ children, ...props }: any) => {
+            const language = props.className ? props.className.replace('lang-', '') : ''
+
+            return (
+              <Box
+                component='code'
+                sx={{
+                  backgroundColor: theme.palette.grey[200],
```

Comment:
We should use a theme colour that works in both light and dark modes. A static colour will not work for both.